docs/docs/Integrations/integrations-mcp.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Integrate Langflow with MCP (Model context protocol)
+slug: /integrations-mcp
+---
+
+Langflow integrates with the [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction). This allows you to use your Langflow flows as tools in other applications that support the MCP protocol, or extend Langflow with the [MCP stdio component](/components-tools#mcp-tools-stdio) to access MCP servers.
```

Comment:
> other applications that support the MCP protocol

I think we want to say "in any MCP client" or "other applications tha timplement the MCP client spec"